org.eclipse.lsp4e.debug/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Debug Adapter client for Eclipse IDE (Incubation)
 Bundle-SymbolicName: org.eclipse.lsp4e.debug;singleton:=true
-Bundle-Version: 0.15.4.qualifier
```

Comment:
This one doesn't need to be changed as you didn't modify it. Only the bundles you change needs to be upversioned (if it wasn't already done since last release)